.github/pull_request_template.md

```diff
@@ -13,12 +13,11 @@ Please delete options that are not relevant.
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
 
-# Checklist:
+## Checklist:
 
 If something is unclear, please submit the PR anyways and ask about what you thought was unclear.
 
-- [ ] This branch is based on `v1.0.0`
-- [ ] This PR is targeted to merge into `v1.0.0` (not `master` or `develop`)
+- [ ] This branch is based on the latest `master`
```

Comment:
I am really confused, aren't we keeping develop until we have the docs site? hahahha

Reply:
Aaah, I misunderstood Renan in the chat. But aren't we publishing docs manually to the site today anyways, why do we need the docs site before we remove develop?

Reply:
Many people still uses the docs on the Github, that is why I think Renan still wants to keep the develop.

IMO, that could be solved with a disclaimer on our mark down, saying that the current version docs is on the site...

Reply:
Updated the Readme with links to the different doc versions, if we push a `latest` tag later we won't have to update the links when we do releases. Maybe we can do that once we release the first final v1.0.0 version.